soc/v2/ti/k3/am6x/Kconfig.soc

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Enphase Energy
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_AM6X
+	bool
+	select SOC_FAMILY_TI_K3
```

Comment:
the `SOC_FAMILY_TI_K3` is not defined anywhere.

Are we missing a Kconfig.soc at this level: https://github.com/zephyrproject-rtos/zephyr/tree/80451dd4eb46233ef2c81c98f48195c650573931/soc/v2/ti/k3 ?

And if yes, should we move the `soc.yml` one level up, and describe the soc family in the soc.yml in addition.

Reply:
Fixed